### ISSUE TITLE
[Urgent!!] Few samples are missing map feature in Scenario test set of Motion Dataset

### ISSUE THREAD

**Issue by SwagJ** [waymo-research/waymo-open-dataset#492]
Hi, 

When I use the test set for submission generation, I found that a few datapoints are missing map_feature, i.e. the length of corresponding map_features attribute of such parsed data are 0. If you iterate all data of test set using loop(counter starting from 0), the index are 12781,13158,15729,22094,22341, 22549,31371, 320235,43960.
Such problem doesn't exist for training and validation set. So, is this void map feature problem in test set a bug or done intentionally?

Also, I noticed that besides the missing map features in few example. Two or three samples also does not have any lanecenters. Is this also done intentionally?

**Comment by scott-ettinger** [comment]
Hi, 
Do you have the scenario IDs for the scenarios you are seeing with missing map data?  This would be helpful for me to look into this issue.

**Comment by SwagJ** [comment]
Hi @scott-ettinger,

Scenario_id are as follow.
  No map_feature cases:  38e2986c8098692f, ab7571715a1d5193, 4c99903f949e8100, a869787ec83d5c8a, 8738e8c0200056fe, 9036b3f956b09fc0, 9465fb15b456855a, ff6686b0e98d66ae, 981f5c4f61505759。
 No LaneCenter cases:  f3d08a52b2188b17, 8d8b3b1d8a32939a, 2bf5f359e847e51c。

Also below is the code for generating above informations.
`data = './waymo/scenario/testing/'
files = os.listdir(data)
dataset = tf.data.TFRecordDataset(
        [os.path.join(data, f) for f in files], num_parallel_reads=1
    )

i = 0
for data in dataset:
    proto_string = data.numpy()
    proto = scenario_pb2.Scenario()
    proto.ParseFromString(proto_string)
    
    scenario_id = proto.scenario_id
    if len(proto.map_features) == 0:
        print(f"Error at {scenario_id}")
    else:
        j = 0
        for map_feature in proto.map_features:
            if getattr(map_feature, 'lane').ByteSize() != 0:
                j += 1
        if j == 0:
            print(f"No centerlines for {scenario_id}")
    i += 1`

**Comment by scott-ettinger** [comment]
Hi, 
Thank you for pointing out this issue. I can confirm that these 12 test scenarios are missing map data. This was an error on our part that was not intentional. This affects about **0.02%** of the test set so it should not significantly affect the metrics results. We will remove these scenarios in the next release. Is your model unable to produce predictions without map features?  Sorry for the trouble. Please let me know if we can help further.

**Comment by SwagJ** [comment]
Hi @scott-ettinger, 

Thank you for confirming this. Well, my model indeed reply on map features to predict trajectory. But only 12 scenarios will should not lead to much of the effect on final result, since there are almost 45k test samples. I should just pack these scenario's prediction with 0 to maintain the integrity of submission file, right?

**Comment by scott-ettinger** [comment]
I would suggest not using all zeros but to either predict objects in their currrent position or extrapolate using constant velocity.

**Comment by SwagJ** [comment]
Hi @scott-ettinger,

Thank you for your reply. I will do so.
